### PR TITLE
[prometheus-node-exporter] allow setting additional host mounts

### PR DESCRIPTION
**Pull request by auhlig**
#### What this PR does / why we need it:

The current version of the chart is limited to `/proc`, `/sys` from the host system.
However, additional mounts from the host system are required by some collectors of the node exporter. For instance systemd wants `/var/run/dbus/system_bus_socket` to be mounted, etc.
This PR allows setting additional volume mounts from the host system via values.

Could you take a look @gianrubio? 

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by cpanato** [comment]
@auhlig please bump the chart version

**Comment by auhlig** [comment]
done @cpanato

File: stable/prometheus-node-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.15.2"
 description: A Helm chart for prometheus node-exporter
 name: prometheus-node-exporter
-version: 0.3.0
+version: 0.3.1
```
Comment (cpanato) [inline stable/prometheus-node-exporter/Chart.yaml]: this adding new functionality I would say to bump to 0.4.0

Comment (auhlig) [reply stable/prometheus-node-exporter/Chart.yaml]: Ah okay. I'll amend to the last commit

**Comment by auhlig** [comment]
Looks like tests passed. Thanks for the review @cpanato

**Comment by davidkarlsen** [comment]
@auhlig please bump the version as another PR got in first - TIA

**Pull request merged**